dotcom-rendering/window.guardian.d.ts

```diff
@@ -1,79 +1,79 @@
 import type {
-	Callback,
-	CMP,
-	ConsentState,
-	VendorName,
-} from '@guardian/consent-management-platform/dist/types';
-import type { WeeklyArticleHistory } from '@guardian/support-dotcom-components/dist/dotcom/src/types';
-import type { WindowGuardianConfig } from './src/model/window-guardian';
-import type { OphanRecordFunction } from './src/web/browser/ophan/ophan';
-import type { DailyArticleHistory } from './src/web/lib/dailyArticleCount';
-import type { ReaderRevenueDevUtils } from './src/web/lib/readerRevenueDevUtils';
+  Callback,
+  CMP,
+  ConsentState,
+  VendorName,
+} from "@guardian/consent-management-platform/dist/types";
+import type { WeeklyArticleHistory } from "@guardian/support-dotcom-components/dist/dotcom/src/types";
+import type { WindowGuardianConfig } from "./src/model/window-guardian";
+import type { OphanRecordFunction } from "./src/web/browser/ophan/ophan";
+import type { DailyArticleHistory } from "./src/web/lib/dailyArticleCount";
+import type { ReaderRevenueDevUtils } from "./src/web/lib/readerRevenueDevUtils";
 
 declare global {
-	/* ~ Here, declare things that go in the global namespace, or augment
+  /* ~ Here, declare things that go in the global namespace, or augment
```

Comment:
Why has this file been formatted with spaces instead of tabs?

Reply:
No idea, but I've undone that.